src/simulators.jl

```diff
@@ -766,15 +772,16 @@ function remd_exchange!(sys::ReplicaSystem{D, G, T},
 end
 
 """
-    simulate_remd!(sys, remd_sim, n_steps; rng=Random.GLOBAL_RNG, n_threads=Threads.nthreads())
+    simulate_remd!(sys, remd_sim, n_steps; rng=Random.GLOBAL_RNG, n_threads=Threads.nthreads(), run_loggers = true)
 
 Run a REMD simulation on a [`ReplicaSystem`](@ref) using a REMD simulator.
 """
 function simulate_remd!(sys::ReplicaSystem,
                         remd_sim,
                         n_steps::Integer;
                         rng=Random.GLOBAL_RNG,
-                        n_threads::Integer=Threads.nthreads())
+                        n_threads::Integer=Threads.nthreads(),
+                        run_loggers = true)
```

Comment:
This should be passed through to the `Threads.@spawn simulate!` call to control those loggers too. With that I think the REMD strategy is okay, i.e. `run_loggers` turns on/off both types of logger. Maybe we could add a `run_exchange_loggers` later if required.